src/Package/Test/DataInspect/REnvironmentProviderTest.cs

```diff
@@ -2,127 +2,162 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Collections;
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 using System.Linq;
-using System.Text;
-using System.Threading;
+using System.Reflection;
 using System.Threading.Tasks;
 using FluentAssertions;
+using Microsoft.Common.Core;
+using Microsoft.Common.Core.Test.Utility;
 using Microsoft.R.Host.Client;
+using Microsoft.R.Host.Client.Session;
 using Microsoft.R.Host.Client.Test.Script;
 using Microsoft.UnitTests.Core.XUnit;
 using Microsoft.VisualStudio.R.Package.DataInspect;
-using Microsoft.VisualStudio.R.Package.Shell;
+using NSubstitute;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace Microsoft.VisualStudio.R.Package.Test.DataInspect {
     [ExcludeFromCodeCoverage]
     [Collection(CollectionNames.NonParallel)]   // required for tests using R Host 
-    public class REnvironmentProviderTest {
-        [Test]
-        [Category.Variable.Explorer]
-        public async Task GetREnvironmentTest() {
-            await RunEnvironmentTestAsync("1", (environments) => {
-                environments.Count.Should().BeGreaterOrEqualTo(2);
-                environments[0].Name.Should().Be(".GlobalEnv");
-                environments[environments.Count - 1].Name.Should().Be("package:base");
-            });
+    public class REnvironmentProviderTest : IAsyncLifetime {
```

Comment:
I see, they are for `REnvironmentProvider`. No, in this case, let's leave them where they are.

---
In reply to: [63224242](https://github.com/Microsoft/RTVS/pull/1770#discussion_r63224242) [](ancestors = 63224242)